### PR TITLE
style: add @progress-info-text-color

### LINKED ISSUE #28980: Progress text color is inconsistent with Progress circle text color
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/ant-design/ant-design/blob/master/components/progress/style/index.less#L96](https://github.com/ant-design/ant-design/blob/master/components/progress/style/index.less#L96)

### Steps to reproduce
https://github.com/ant-design/ant-design/blob/master/components/progress/style/index.less#L96 text color seems inconsistent with circle text color ([this](https://github.com/ant-design/ant-design/blob/master/components/progress/style/index.less#L166)). Should them be same? 

### What is expected?
- Text color uses same variables
- Or there are different variables respectively `progress-text-color` and `progress-circle-text-color`, as similar as `progress-text-font-size`

### What is actually happening?
Cannot specify normal progress text color

| Environment | Info |
|---|---|
| antd | 4.10.3 |
| React | React |
| System | Mac |
| Browser | Chrome |

### PR DESCRIPTION
**Pull request by yuxuan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
fix https://github.com/ant-design/ant-design/issues/28980

### 💡 Background and solution

Fix issue: https://github.com/ant-design/ant-design/issues/28980

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    |
| 🇨🇳 Chinese | 新增  `@progress-info-text-color` |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

Not sure this change is appropriate. The Progress text color changed from `@text-secondary-color` to `@text-color`, which may cause visual difference.

### CODE REVIEW

File: components/progress/style/index.less [hunk 1]
```diff
@@ -93,7 +93,7 @@
     display: inline-block;
     width: 2em;
     margin-left: 8px;
-    color: @text-color-secondary;
+    color: @progress-text-color;
```
Comment (xrkffgg) [inline components/progress/style/index.less:96]: 把 style 里的 改成这个

Comment (yuxuan) [reply components/progress/style/index.less:96]: 这个在`circle`里面用到了，是`@text-color`，两个互斥

Comment (xrkffgg) [reply components/progress/style/index.less:96]: 那换个名字

File: components/progress/style/index.less [hunk 2]
```diff
@@ -163,7 +163,7 @@
     width: 100%;
     margin: 0;
     padding: 0;
-    color: @progress-text-color;
+    color: @progress-circle-text-color;
```
Comment (xrkffgg) [inline components/progress/style/index.less:166]: 原有的不动，上面的新增个。不然会造成破坏性更新的。

**Comment by xrkffgg** [comment]
cc

File: components/style/themes/default.less [hunk 3]
```diff
@@ -532,10 +532,11 @@
 // --
 @progress-default-color: @processing-color;
 @progress-remaining-color: @background-color-base;
-@progress-text-color: @text-color;
+@progress-info-text-color: @text-color-secondary;
```
Comment (afc163) [inline components/style/themes/default.less:535]:
```suggestion
@progress-info-text-color: @progress-text-color;
```

否则修改 @progress-text-color 会失效。

**Pull request merged**